Implement: Show widget form controls initially in collapsed state

johnregan3 opened the pull request:
Adds accordion-style tabs to hide/show widget interface.  Causes jumpiness when widget tab expands down the page causing the scrollbar to appear.  Not sure what we can do about this without forcing the scrollbar at all times.

Fixes #7

johnregan3 committed:
Widget Accordion Tabs

Commit:
Changing Tabs to slideToggle()

westonruter commented on class-widget-form-wp-customize-control.php:
```diff
@@ -24,35 +24,45 @@ public function render_content() {
+					</h4>
+				</div>
+			</div>
+			<!-- need to remove display:block -->
```
@johnregan3 is it removed?

johnregan3 replied:
@westonruter Geez, yeah.  I just forgot to remove my note to remove that.  Ironic, huh?

Commit:
widget-accordion Removing unnecessary note

westonruter committed:
Scope controlTabs to control container

Click event handlers from other controls were effecting each other.
Result was the widget could not expand if there was an even number
of widgets in the sidebar, since they would all cancel each other out.

Use toggle code as found in WordPress core's widgets.js

westonruter committed:
Add live in-widget-title

Normalize JS methods to be camelCase

westonruter commented:
Since the widgets are collapsed by default, and since there could be any number of Text widgets (for example), it's important that the `in-widget-title` be displayed. So I added support for that in 7d31496:

![customize_twenty_eleven__wordpress_and_7 _bash_and_all_items_-_google_drive](https://f.cloud.github.com/assets/134745/1155750/cdf6e0ee-1f76-11e3-8473-9d875de27d28.png)
